src/main/java/org/spongepowered/api/entity/hanging/Hanging.java

```diff
@@ -43,7 +43,7 @@
      * Sets the facing direction of this entity.
      *
      * @param direction The direction to face
-     * @param forced Whether physics is ignored
+     * @param forced    Whether physics is ignored
```

Comment:
The rules apply to us, but unlike PRs from non-team members, I'm free to do a cleanup pass on this later on when there aren't more important things that need to be handled first. Saying "oh well, let's keep this important PR on ice just because it's missing a few lines of javadoc" is stupid when it can be merged, people can get to work on things, then next week when I have time I can fix the formatting issues in one clean pass.

Asking a team member to do that on the third-party PR, or accepting two PRs for every PR (one for functionality and another to clean up) isn't realistic and it's much easier to ensure that PRs are "clean" before they're pulled. This is not the case here, I have more pressing stuff to be spending time working on, the javadoc and the little code style issues can wait.